VCFoodDeliveryApp/Migrations/20210525160139_AddCartForUser.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace VCFoodDeliveryApp.Migrations
+{
+    public partial class AddCartForUser : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.CreateTable(
+                name: "Cart",
+                schema: "Identity",
+                columns: table => new
+                {
+                    Id = table.Column<string>(type: "nvarchar(450)", nullable: false),
+                    ApplicationUserId = table.Column<string>(type: "nvarchar(450)", nullable: false),
```

Comment:
You are correct this shouldn't be nullable. I can't think of a valuable reason for it to be.  🤔. 

Great catch!